index.html

```diff
@@ -9298,6 +9298,261 @@
+               <code><a>RTCQuicTransport</a></code>'s <code><a>onerror</a></code>.</div></li>
+              <table class="parameters">
+                <tbody>
+                  <tr>
+                    <th>Parameter</th>
+                    <th>Type</th>
+                    <th>Nullable</th>
+                    <th>Optional</th>
+                    <th>Description</th>
+                  </tr>
+                  <tr>
+                    <td class="prmName">data</td>
+                    <td class="prmType"><code>Uint8Array</code></td>
+                    <td class="prmNullFalse"><span role="img" aria-label=
+                    "False">&#10008;</span></td>
+                    <td class="prmOptFalse"><span role="img" aria-label=
+                    "False">&#10008;</span></td>
+                    <td class="prmDesc"></td>
+                  </tr>
+                </tbody>
+              </table>
+              <div>
+                <em>Return type:</em> <code>void</code>
+              </div>
+            </dd>
+            <dt><dfn><code>read</code></dfn></dt>
+            <dd>
+              <p>Reads from <code><a>RTCQuicStream</a></code>. When the <code>read()</code> method is called,
+              the user agent <em class="rfc2119" title="MUST">MUST</em> run the following
+              steps:</p>
+              <p>1. Let <var>stream</var> be the <code><a>RTCQuicStream</a></code> object on which <code>read()</code>
+              is invoked.</p>
+              <p>2. If <var>stream</var>'s <code>state</code> is <code>closing</code> or <code>closed</code>, then abort
+              these steps.</p>
+              <p>3. If <var>stream</var>'s <a>[[\Readable]]</a> slot is <code>false</code>, <a>throw</a> an
+              <code>InvalidStateError</code>.</p>
+              <div>
+                <em>No parameters.</em>
+              </div>
+              <div>
+                <em>Return type:</em> <code>Uint8Array</code>
+              </div>
+            </dd>
+        </dl>
+      </section>
+      </div>
+    </section>
+    <section id="rtcquicstreamstate*">
+      <h2><dfn>RTCQuicStreamState</dfn> Enum</h2>
+      <p>The <code>RTCQuicStreamState</code> provides information
+      on the state of the QUIC stream.</p>
+      <div>
+        <pre class="idl">enum RTCQuicStreamState {
+    "idle",
+    "open",
+    "closing",
+    "closed"
+};</pre>
+        <table data-link-for="RTCQuicStreamState" data-dfn-for="RTCQuicStreamState"
+        class="simple">
+          <tbody>
+            <tr>
+              <th colspan="2">Enumeration description</th>
+            </tr>
+            <tr>
+              <td><dfn><code id=
+              "idl-def-RTCQuicStreamState.idle">idle</code></dfn></td>
+              <td>
+                <p>The <a>user agent</a> is attempting to establish the underlying stream
+                transport. This is the initial state of an
+                <code><a>RTCQuicStream</a></code> object.</p>
```

Comment:
So, it sounds like this API is designed such that you can call `createStream` at any time, even if the QUIC transport isn't ready? This makes sense, since it's convenient and matches `createDataChannel`. Just want to clarify.

Reply:
Yes, that was the intent. quicTransport.createStream() provides an RTCQuicStream quicStream with state initialized to "idle" and quicTransport.onstream Event Handler will event an RTCQuicStream with a state of "open".  So quicTransport.state need not be  "connected" when quicTransport.createStream() is called;  once quicStream.transport.state transitions to "connected", then quicStream.state can transition to "open".  Does this make sense?